flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -117,8 +119,88 @@
                     .withDescription(
                             "The set of variables that should be excluded for the reporter named <name>. Only applicable to tag-based reporters (e.g., PRometheus, InfluxDB).");
 
+    private static final Description.DescriptionBuilder addFilterDescription(
+            Description.DescriptionBuilder description) {
+        return description
+                .text("Filters are specified as a list, with each filter following this format:")
+                .linebreak()
+                .text("%s", code("<scope>[:<name>[;<name#N>][:type[;type#N]]]"))
+                .list(
+                        text(
+                                "scope: Filters based on the logical scope.%s"
+                                        + "Specified as a pattern where %s matches one or more characters and %s separates scope components.%s"
```

Comment:
Without a line break it becomes too much of a mess imo. 2 line breaks worked better, but looked a bit because there now was an empty line within the paragraph, but none at the end (i.e., between list items).
I now added 2 more linebreaks to the end of each list item:

![image](https://user-images.githubusercontent.com/5725237/161026317-28c972a9-fe73-47e8-9806-a04cf30b7dd0.png)

Reply:
looks good 👍